godtools/Settings/GTConstants.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  GTConstants.swift
+//  godtools
+//
+//  Created by Devserker on 4/19/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct GTConstants {
```

Comment:
One idea I've done in the past which is nice is to create an extension for UIColor and put the apps custom colors in it. The result is we could do UIColor.gtWhite or UIColor.gtBlue (or in swift:   `.gtWhite`) Can you try this?

If this doesn't work, can we group colors into a separate struct in this file? I think that will help it stay organized better.